components/server/src/orgs/organization-service.ts

```diff
@@ -497,6 +508,12 @@ export class OrganizationService {
         if (settings.defaultRole) {
             result.defaultRole = settings.defaultRole;
         }
+        if (settings.timeoutSettings) {
+            result.timeoutSettings = {
+                allowChangeByMembers: settings.timeoutSettings.allowChangeByMembers ?? true,
```

Comment:
```suggestion
                allowChangeByMembers: settings.timeoutSettings.allowChangeByMembers,
```

Reply:
Oh, I thought pushing the default of `true` is what you meant in here: https://github.com/gitpod-io/gitpod/pull/20099#discussion_r1711443860 👀. It should make sure we don't ever return settings with an `undefined` value.

Reply:
Hm. Have to take a closer look at this :eyes:

Reply:
@filiptronicek I now have a suggestion here: e9764bc3a9b0ffee32f7a459ca601e4a56c6c8ab

Basically renaming the flag to `denyUserTimeouts` (happy about other suggestions) avoids all the translation/initialization fuzz. :slightly_smiling_face: WDYT? (Also, simplified some of the checks in react)

I wanted to do it differently first, but ran into problems with the way we handle DB shapes... oh well. :see_no_evil:

Reply:
Also, is it by design that a user can't change the default timeout, but deny members their custom timeouts, if the plan is not paid? :thinking:

Reply:
Applied your changes to this branch